package.json

```diff
@@ -18,6 +18,7 @@
     "@fortawesome/free-solid-svg-icons": "^6.1.1",
     "@fortawesome/react-native-fontawesome": "^0.2.7",
     "@hathor/wallet-lib": "^1.0.1",
+    "@jest/globals": "^29.7.0",
```

Comment:
This should be a dev-dependency, right?

Reply:
I checkout the commit `@8120e600` and verified that @jest/globals is a dependency for jest-runtime, which is already a dependency for jest:

```sh
❯ npm ls \@jest/globals
HathorMobile@0.26.0-rc6 /Users/alexrusenhack/repos/hathor/hathor-wallet-mobile
└─┬ jest@29.2.1
  └─┬ @jest/core@29.7.0
    └─┬ jest-runtime@29.7.0
      └── @jest/globals@29.7.0
```

I'm removing this commit 591c6af71425e0e6a9cbf90bc0bdf2fcb95b8021